great_expectations/expectations/metrics/column_map_metrics/column_value_lengths.py

```diff
@@ -146,6 +160,55 @@ def _pandas(
 
         return metric_series
 
+    @column_condition_partial(engine=PolarsExecutionEngine)
+    def _polars(
+        cls,
+        column,
+        _metrics,
+        min_value=None,
+        max_value=None,
+        strict_min=None,
+        strict_max=None,
+        **kwargs,
+    ):
+        column_lengths, _, _ = _metrics.get(
+            f"column_values.value_length.{MetricPartialFunctionTypeSuffixes.MAP.value}"
+        )
+
+        metric_series = None
+        if min_value is not None and max_value is not None:
+            if strict_min and strict_max:
+                metric_series = (column_lengths > min_value) & (
+                    column_lengths < max_value
+                )
+            elif strict_min and not strict_max:
+                metric_series = (column_lengths > min_value) & (
+                    column_lengths <= max_value
+                )
+            elif not strict_min and strict_max:
+                metric_series = (column_lengths >= min_value) & (
+                    column_lengths < max_value
+                )
+            elif not strict_min and not strict_max:
+                metric_series = (column_lengths >= min_value) & (
+                    column_lengths <= max_value
+                )
+        elif min_value is None and max_value is not None:
+            if strict_max:
+                metric_series = column_lengths < max_value
+            else:
+                metric_series = column_lengths <= max_value
+        elif min_value is not None and max_value is None:
+            if strict_min:
+                metric_series = column_lengths > min_value
+            else:
+                metric_series = column_lengths >= min_value
```

Comment:
If this logic appears in multiple metrics I'd love to refactor it into a central place e.g. with the execution engine.

Reply:
Great idea